shared/continuations/include/LgcCpsDialect.h

```diff
@@ -0,0 +1,44 @@
+/*
+************************************************************************************************************************
+*
+*  Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+*
+************************************************************************************************************************
+*/
```

Comment:
Use the full copyright header used on gpuopen with the license text; also for the .td and .cpp files.